src/modules/simulator/simulator_mavlink.cpp

```diff
@@ -434,6 +434,25 @@ void Simulator::handle_message(mavlink_message_t *msg, bool publish)
 
 		break;
 
+	case MAVLINK_MSG_ID_IRLOCK_REPORT:
```

Comment:
This needs to be replaces by `MAVLINK_MSG_ID_LANDING_TARGET` as we spoke of.

Reply:
https://github.com/PX4/sitl_gazebo/pull/135